scripts/mkcloud

```diff
@@ -479,6 +479,14 @@ function crowbar_register
     done
 }
 
+# setup an NFS server on the first lonely node
+function lonelynode_nfs_server
+{
+    local nfsserver=($(nodes ids lonely))
```

Comment:
Actually assigning that to nfsserver will just give the first element of that list. There might be better ways to do it. I am open the suggestions.